### ISSUE TITLE
Unable to determine android gradle version

### ISSUE THREAD

**Issue by Cherubyx** [bitrise-steplib/steps-install-missing-android-tools#17]
Ever since today's release with 2.0.0 all of my builds have been failing. I'm reverting to 1.1.0 for now just so we can build.

We use a secondary gradle file to put all of our dependencies and versioning. We would like to keep it this way (many reasons). The step only seems to verify via regex if it exactly written out in a single line in the dependencies bracket.

**My project gradle file:**
```
buildscript {
  ...
  dependencies {
    classpath deps.gradle
  }
}
```
**Secondary gradle file:**
```
ext.versions = [
        ...
        'gradleVersion' : '2.3.3',
        ...
]

ext.deps = [
        ...
        'buildGradle' : "com.android.tools.build:gradle:${versions.gradleVersion}",
        ...
]
```

**I tried out this below and this works as expected, but this is what we don't want to use:**
```
buildscript {
  ...
  dependencies {
    classpath 'com.android.tools.build.gradle:2.3.3'
  }
}
```
**Log file:**
```
+------------------------------------------------------------------------------+

| (3) install-missing-android-tools@2.0.0                                      |
+------------------------------------------------------------------------------+
| id: install-missing-android-tools                                            |
| version: 2.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2017-08-23T15:30:33Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
INFO[15:30:33] Start installing (golang) with apt-get       
INFO[15:30:33]  * [OK] Step dependency (go) installed, available. 
Configs:
- RootBuildGradleFile: build.gradle
- GradlewPath: ./gradlew
- AndroidHome: /opt/android-sdk-linux
Step determined configs:
- root build.gradle file: /bitrise/src/build.gradle
Failed to determine android gradle plugin version, error: missing android gradle plugin version
|                                                                              |
+---+---------------------------------------------------------------+----------+
| x | install-missing-android-tools@2.0.0 (exit code: 1)            | 2.98 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: ...bitrise-steplib/steps-install-missing-android-tools/issues |
| Source: ...://github.com/bitrise-steplib/steps-install-missing-android-tools |
+---+---------------------------------------------------------------+----------+
```

**Comment by fmedlin** [comment]
Had the same issue today. Once I reverted to v1.1.0 builds completed successfully again.

**Comment by senthilkumarv** [comment]
The new Regex (main.go:84) to match gradle version seem to fail for beta/alpha version of android gradle plugin. This one works
```\s*classpath\s*'com.android.tools.build:gradle:(?P<version>[0-9\-a-z.*]*)'\s*```

**Comment by godrei** [comment]
Hi guys, thanks for the reports!

We released a new version ([2.0.1](https://github.com/bitrise-steplib/steps-install-missing-android-tools/releases/tag/2.0.1)), which does not use the Android Gradle Plugin version anymore.

I'll close this issue, but if you need more info/help just let us know and we'll reopen it!
